frontend/msSettingsManager.pas

```diff
@@ -480,6 +482,37 @@ procedure TSettingsManager.BuildFromPluginsItemClick(Sender: TObject);
   slSelection.Free;
 end;
 
+procedure TSettingsManager.SelectSimilarNodesItemClick(Sender: TObject);
+var
+  i, index: Integer;
+  tvNode: TTreeNode;
+  reqSmashType, currentSmashType: TSmashtype;
+  slSelection: TStringList;
+begin
+  slSelection := TStringList.Create;
+  try
+    for i := 0 to Pred(tvRecords.SelectionCount) do
+    begin
```

Comment:
Move this up the pevious line?  I personally don't follow the standard Delphi/Pascal spec of putting "begin" on a new line in most of my code, simply because I think it's a waste of a line and prefer treating `begin ... end` as `{ ... }`.